### DISCUSSION TITLE
Guide Dogs Of The Metaverse

### DISCUSSION THREAD

**Discussion by gregm-mwb** [omigroup/omigroup#153] [Ideas]
The metaverse is the next evolution of the internet and already interaction is superior to the web. It's important that the metaverse is inclusive from the start and does not leave people with disability isolated. I believe we need to reinvent the way we guide disabled people through interactive worlds.

One of the most important aspects of making the metaverse accessible is a cross-compatible avatar. With so many platforms arising in the metaverse the compatibility of an avatar is the first step towards accessibility. A standardised set of rules could be made for avatars which would allow the same avatar to travel through different platforms and interact in the same way. Once that is achieved then a set of guidelines could be made that determine the labeling programmatically of a scene in the metaverse so that people, objects, and terrain can be interpreted.

Once cross-compatibility of the avatar has been implemented then a new way of interacting with the metaverse should be made. Just as screenreaders navigated the internet an "AI Guide Dog" could navigate the metaverse. The AI dog would interpret the scene and handle the navigation of the metaverse. It could describe objects, groups of people, and the terrain and architecture of the scene without overloading the user with information. It could accept commands to navigate the scene and interact with objects.

**Comment by mrmetaverse** [comment]
This is both a cute idea, and very important one. Inclusion and accessibility are paramount to getting the next iteration of the web right. I love the idea of an AI Guide Dog assisting Metaverse explorers. One thing that comes to mind though is the creative control of those who are designing and shaping Worlds / experiences. Interoperable avatars may work for some, but not for others. For example, if someone created a game like Half Life, they may not want your toon character in their World. It might clash with the design aesthetic or overall immersion they are going for. So just because we _can_ have interoperable avatars, that doesn't always mean that we will or should. 

We want to be mindful of creators. If we impose to strict of rules, or standards, some creators will reject the effort entirely. You can have the best protocol in the World, but it is useless if nobody adopts it. Therefore, I do get nervous about hinging the entire success of a plan on one piece of interoperability. Whether its avatars, inventory, or traversal. Some creators will not want it in their creations.
